universal/include/userver/utils/strong_typedef.hpp

```diff
@@ -365,7 +365,7 @@ std::string ToString(const StrongTypedef<Tag, std::string, Ops>& object) {
 }
 
 template <typename Tag, typename T, StrongTypedefOps Ops,
-          typename = std::enable_if_t<meta::kIsInteger<T>>>
+          typename = std::enable_if_t<meta::kIsNumber<T>>>
```

Comment:
Add some tests on new functionality. A sample from the PR description could be used for that

Reply:
Done